examples/v1/general-purpose.yaml

```diff
@@ -9,8 +9,12 @@ metadata:
 spec:
   disruption:
     consolidateAfter: 0s
+    expireAfter: Never
```

Comment:
In v1 `expireAfter` is in the template (and already there)

Reply:
Yea, just removed